README.md

```diff
@@ -1,2 +1,67 @@
 # Pole-Climbing-Robot
 Srishti 2020
+
+![Final Product](https://github.com/DanielLarusso1/Pole-Climbing-Robot/blob/master/Images%20and%20Videos/Images/Robot%20CAD.jpeg)
```

Comment:
As you have moved Robot CAD.jpeg out of Images folder, update this URL with new URL of Complete Bot Design.jpeg